#DoCoMo QA API Request message
Header header
string text 
